client/packages/common/src/utils/quantities/quantities.test.ts

```diff
@@ -1,39 +1,65 @@
 import { QuantityUtils } from './quantities';
 
-describe('QuantityUtils - suggested quantity', () => {
-  it('is defined', () => {
-    expect(QuantityUtils.suggestedQuantity).toBeDefined();
-  });
+describe('QuantityUtils', () => {
```

Comment:
Ah recommend `hide whitespace` here - only actual change is adding the tests for the two new methods, at the bottom of the file